### ISSUE TITLE
Transfer Ticket to other user

### ISSUE THREAD

**Issue by inbaz** [inbaz/ers#394]
From #198 

Transfer ticket
1) Purchaser: emails prereg@eja.net with request to transfer a ticket.
2) preregco: Ask which ticket. and Name, Birthday, email, Nationality of
the new participant (e.g. the add Person form).
3) preregco: Add the person to the ticket.
4) ERS: Send notification email to old participant
5) ERS: Send new E-ticket to new participant and purchaser"

**Comment by inbaz** [comment]
For documentation purposes I'd like to do a cancellation in the background and generate a new item for the new participant. Maybe the old items can be held in status "transferred" instead of cancelled.

**Comment by smadamit** [comment]
Don't forget #328. Duplicate? -> merge it...

**Comment by smadamit** [comment]
**decision in #329:** After the transfer of the tickets Alice and Bob (old owner of the tickets) get an info email that the purchaser transferred the tickets to a new person. If Alice and Bob are fine with that -> OK. Otherwise: They can contact prereg@eja.net

In the end: it's a problem between the guys that pay together

**Comment by inbaz** [comment]
There are two was how the data of the receiving person get into the system:
1. the receiving person just registers on the platform (will maybe be enabled with the profile functions)
2. the prereg coordinator asks the receiving person for Firstname, Surname, date of birth and an email address.

There is one major problem: If the former buyer already paid the ticket the payment was received with the old order code. If the receiving person wants to refund the ticket it's getting complicated. We need to think about how transfered items are stored and how they are connected.

**Comment by smadamit** [comment]
Why not give a new order code for transferred tickets. The transfer could be a kind of an order.

**Comment by inbaz** [comment]
If the new Participant is setup wrong and we adjust i.e. the date of birth, the order gets marked as unpaid which will be never be in paid status again. Because the matching needs to be made manual but there is a different code, then, a different name and nothing which leads to the correct order.

I like the idea of a new order, but the matching needs to be made in any way. I do have an idea to add a transferred_item_id field in the the item table of the database. With that field I'm able to follow the trace until I find an item without the transferred_item_id which is of interest.

**Comment by smadamit** [comment]
I suggest -> workaround for this year -> I put it to after EJC

**Comment by smadamit** [comment]
We decided, that we need this feature for this year. As for all features this year, only in the admin panel. Profile features will come 2016.

We also decided, that ticket transfers are only allowed within the same agegroup. So a week ticket of the first period for a youth for 90 € can later only be transferred to a new person which is also a youth. A transfer to an adult person is not possible. That makes life for the prereg-coordinator and the ERS-team much easier and it's even better for the juggler, because the rule "treats all jugglers equal".
